static/docs/get-started/import-data.md

```diff
@@ -0,0 +1,69 @@
+# Import Data
+
+We've seen how to [retrieve](/doc/get-started/retrieve-data) data
+[stored](/doc/get-started/store-data) for a single <abbr>DVC project</abbr>.
+`dvc import` also lets us download <abbr>data artifacts</abbr>, but from a
+different project that is hosted on a Git repository, and creates an import
+stage (DVC-file) to track changes in this
+[external dependency](/doc/user-guide/external-dependencies).
+
+For example, in the [Add Files](/doc/get-started/add-files) chapter we download
+raw data using the `dvc get` command – which unlike `dvc import` does not create
+an import stage. Let's say we wanted to download the same dataset, while keeping
+the connection to its source project:
```

Comment:
To be honest I looks like both paragraphs are saying more or less the same. Can we try to merge them?

Also a few comments:

`download data for a single DVC project` - not clear enough is it for that project or is it data defined/tracked in that project? 

first paragraph is complicated and confusing still :( took me a while to understand that it's about push/pull. Can we just write something like this:

`We've see how to use push and pull to store and download back data for a project. But what if we want to plug some data or ML model produced in one project into another one? To enable reusability of data artifacts.

One option, of course is to download it (wget or `dvc get`) and use `dvc add`. But this way the connection is lost, we don't know where that data came from, how it was produced, are there new versions of it, etc. `dvc import` similar to what `dvc get` (or wget/curl) + `dvc add` can do but it also saves some additional context into the DVC-file it creates:
``` 

`import stage` - let's try to avoid this terminology here - just a DVC-file that has some addition information inside

Reply:
OK. I did an almost complete rewrite based in 6126652 on these comments. What do you think?